src/boot/efi/util.h

```diff
@@ -27,16 +27,15 @@
 #define xnew_alloc(type, n, alloc)                                           \
         ({                                                                   \
                 UINTN _alloc_size;                                           \
-                if (__builtin_mul_overflow(sizeof(type), (n), &_alloc_size)) \
-                        assert_not_reached();                                \
+                assert_se(!__builtin_mul_overflow(sizeof(type), (n), &_alloc_size)); \
```

Comment:
The ocd in me is complaining that the remaining `\` arent't aligned.

Reply:
This is how emacs aligns it, and emacs is the law, don't you know? ;-)

i.e. it aligns \ on the same column over the whole macro, except when lines are above some threshold, in which case it puts it right after the end of the line. This threshold is hit here...

Reply:
Blurgh. An editor you have learn to use and is efficient on resources? You should go with the flow and use an Electron powered IDE. :stuck_out_tongue: